src/survey.ts

```diff
@@ -0,0 +1,212 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ----------------------------------------------------------------------------
+
+import { commands, MessageItem, Uri, window, workspace } from 'vscode';
+import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azureextensionui";
+import { configPrefix } from './constants';
+import { ext } from "./extensionVariables";
+import { assert } from './fixed_assert';
+import { httpGet } from "./httpGet";
+import { hoursToMs, minutesToMs, weeksToMs } from "./util/time";
+
+namespace stateKeys {
```

Comment:
>stateKeys [](start = 10, length = 9)

Nit: should we call this surveryStateKeys to disambiguate from possible other stateKeys we may have?

Reply:
It's limited to this file so not an issue.